src/ansys/mapdl/core/mapdl_core.py

```diff
@@ -2272,6 +2276,24 @@ def __del__(self):
                 except Exception:
                     pass
 
+    def _cleanup_loggers(self):
+        """Clean up all the loggers"""
+        # Detached from ``__del__`` for easier testing
+        logger = self.logger
```

Comment:
Same question here. Is it done on purpose?

Reply:
Inside this function, it is not relevant because we do not use the other ``logger``.